internal/minioclient/client.go

```diff
@@ -142,20 +156,24 @@ func (client *Minio) Download(bucket, objectName string, buffer []byte) ([]byte,
 }
 
 // Delete deletes object.
-func (client *Minio) Delete(bucket, objectName string) error {
-	err := client.API.RemoveObject(bucket, objectName)
+func (client *Minio) Delete(bucket, objectName string) (err error) {
+	defer mon.Task()(nil)(&err)
+
+	err = client.API.RemoveObject(bucket, objectName)
 	if err != nil {
 		return MinioError.Wrap(err)
 	}
 	return nil
 }
 
 // ListObjects lists objects.
-func (client *Minio) ListObjects(bucket, prefix string) ([]string, error) {
+func (client *Minio) ListObjects(bucket, prefix string) (names []string, err error) {
+	defer mon.Task()(nil)(&err)
+
 	doneCh := make(chan struct{})
 	defer close(doneCh)
 
-	names := []string{}
+	names = []string{}
```

Comment:
this can be removed as well now that it's defined as a return variable (same as other comment)

Reply:
done